pages/careers/_human-friendly.html

```diff
@@ -19,15 +19,17 @@ <h3>How We Work</h3>
     <h3>Benefits</h3>
     <p>Our benefits reflect our values. We believe compensation should be fair, transparent, and generous. We hire Grunts in many countries, so some details may vary.</p>
     <ul>
-      <li><strong>Above-Market Salary.</strong> To reduce bias and increase transparency, we compute all salaries <a href="https://blog.gruntwork.io/how-we-hire-at-gruntwork-42193d583e24">using formulas</a>. The formula factors in your title and location and uses a multiplier to produce a result that's above market for that title and location.</li>
+      <li><strong>Location Independent Above-Market Salary.</strong> To reduce bias and increase transparency, we compute all salaries <a href="https://blog.gruntwork.io/how-we-hire-at-gruntwork-42193d583e24">using formulas</a>. The formula factors in your title and uses a multiplier to produce a result that's above market for that title. Our salaries are location independent.</li>
```

Comment:
```suggestion
      <li><strong>Location Independent, Above-Market Salary.</strong> To reduce bias and increase transparency, we compute all salaries <a href="https://blog.gruntwork.io/how-we-hire-at-gruntwork-42193d583e24">using formulas</a>. The formula factors in your title and uses a multiplier to produce a result that's above market for that title. Our salaries are location independent.</li>
```

Reply:
Fixed in https://github.com/gruntwork-io/gruntwork-io.github.io/pull/684/commits/0f144aaa4f9c31160c31ca073c20c20a9e483fa4.